src/score-bot/components/report-item/report-item.tsx

```diff
@@ -1,67 +1,52 @@
 import { renderStyledComponentToString } from "../../../shared/utilities/render-styled-component-to-string";
 import { FeedbackReport } from "./feedback-report";
 import { getLastFeedback, getLastScore, getMaxScore, getNumberOfAttempts, isFeedbackOutdated } from "../../utils";
 
-interface Props {
-  initMessage: IReportItemInitInteractive;
-}
-
-export const ReportItemComponent: React.FC<Props> = (props) => {
-  useReportItem<IInteractiveState, IAuthoredState>({
-    metadata: {
-      compactAnswerReportItemsAvailable: true
-    },
-    handler: request => {
-      const { interactiveState, authoredState, platformUserId, version, itemsType } = request;
-
-      if (!version) {
-        // for hosts sending older, unversioned requests
-        // tslint:disable-next-line:no-console
-        console.error("ScoreBOT Report Item Interactive: Missing version in getReportItemAnswer request.");
-      }
-      else if (semver.satisfies(version, "2.x")) {
-        const items: IReportItemAnswerItem[] = [];
-
-        if (itemsType === "fullAnswer") {
-          // This item intentionally doesn't include answer text. Answer text is obtained by portal report itself
-          // and preloaded while it's waiting for report items response.
-          items.push({ type: "answerText" });
-
-          const score = getLastScore(interactiveState);
-          const maxScore = getMaxScore(authoredState);
-          const attempts = getNumberOfAttempts(interactiveState);
-          const feedback = getLastFeedback(authoredState, interactiveState);
-          const isOutdated = isFeedbackOutdated(interactiveState);
-          const feedbackHtml = renderStyledComponentToString(
-            <FeedbackReport score={score} maxScore={maxScore} attempts={attempts} feedback={feedback} outdated={isOutdated} />,
-          );
-          items.push({ type: "html", html: feedbackHtml });
-        }
-
-        if (itemsType === "compactAnswer") {
-          const score = getLastScore(interactiveState);
-          const maxScore = getMaxScore(authoredState);
-          if (score !== null && maxScore !== null) {
-            items.push({ type: "score", score, maxScore });
-          }
-        }
+export const reportItemHandler: IGetReportItemAnswerHandler<IInteractiveState, IAuthoredState> = request => {
+  const { interactiveState, authoredState, platformUserId, version, itemsType } = request;
+
+  if (!version) {
+    // for hosts sending older, unversioned requests
+    // tslint:disable-next-line:no-console
+    console.error("ScoreBOT Report Item Interactive: Missing version in getReportItemAnswer request.");
+  }
+  else if (semver.satisfies(version, "2.x")) {
+    const items: IReportItemAnswerItem[] = [];
+
+    if (itemsType === "fullAnswer") {
+      // This item intentionally doesn't include answer text. Answer text is obtained by portal report itself
+      // and preloaded while it's waiting for report items response.
+      items.push({ type: "answerText" });
```

Comment:
this comment is confusing. It says the item doesn't include answer text and then adds the answerText item.  Perhaps the intention is that `html` item that is added doesn't include the answer text since it will be added based on the `answerText` item.

Reply:
So, what I meant in this comment is that I'd typically expect this line to look like this:
```
items.push({ type: "answerText", answerText: interactiveState.answerText });
```
but instead, we push an "empty" item (just type, no content):
```
items.push({ type: "answerText" });
```
I wanted to clarify that the content of the answer text is calculated by the portal-report itself, as it was confusing for me at the beginning (I think we discussed that on slack one day).

So, any suggestion for better wording here so the comment doesn't make it even worse? :)